.github/workflows/playwright.yml

```
@@ -0,0 +1,41 @@
+name: Playwright Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-playwright:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '17.x'
```

Node 17 is no longer officially supported. The current LTS version is 16, the current dev version is 18. Let's go with 18 here?

```suggestion
          node-version: '18.x'
```